repos/system_upgrade/el7toel8/actors/checkwireshark/actor.py

```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.models import Report
+from leapp.reporting import create_report
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+
+
+class CheckWireshark(Actor):
+    """
+    Report a couple of changes in tshark usage
+    """
+
+    name = 'check_wireshark'
+    consumes = (InstalledRedHatSignedRPM, )
+    produces = (Report, )
+    tags = (FactsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+		if has_package(InstalledRedHatSignedRPM, 'wireshark'):
+			create_report([
+				reporting.Title('Changed cli option'),
```

Comment:
Based on the title, it is not clear about what the report is. From the info it seems that this is about *tshark* utility. So something like: *tshark: changed cli options*. same below.